packages/@storefront/structure/src/filtered-list/index.ts

```diff
@@ -7,14 +7,21 @@ class FilteredList {
   refs: { filter: HTMLInputElement };
   props: FilteredList.Props = {
     items: [],
+    paginate: true,
   };
   state: FilteredList.State = {
     items: [],
   };
 
   childProps() {
```

Comment:
I had thought of that, but I didn't because I realized we aren't actually bound to `PagedList.Props` if someone overrides the structure. But perhaps we are, so I think explicitly typing it is still correct.

Reply:
I meant `childProps(): PagedList.Props`

Reply:
But this is fine